### ISSUE TITLE
Getting a IOERROR with my condarc

### ISSUE THREAD

**Issue by bkreider** [conda/conda#3066]
I'm having trouble figuring out what is malformed.  It would be helpful to have this handled and report what is broken.

```
Using Anaconda Cloud api site https://api.anaconda.org
Fetching package metadata: An unexpected error has occurred, please consider sending the
following traceback to the conda GitHub issue tracker at:

    https://github.com/conda/conda/issues

Include the output of the command 'conda info' in your report.


Traceback (most recent call last):
  File "/opt/wakari/anaconda/bin/conda", line 5, in <module>
    sys.exit(main())
  File "/opt/wakari/anaconda/lib/python2.7/site-packages/conda/cli/main.py", line 173, in main
    args_func(args, p)
  File "/opt/wakari/anaconda/lib/python2.7/site-packages/conda/cli/main.py", line 180, in args_func
    args.func(args, p)
  File "/opt/wakari/anaconda/lib/python2.7/site-packages/conda/cli/main_create.py", line 49, in execute
    install.install(args, parser, 'create')
  File "/opt/wakari/anaconda/lib/python2.7/site-packages/conda/cli/install.py", line 240, in install
    prefix=prefix)
  File "/opt/wakari/anaconda/lib/python2.7/site-packages/conda/cli/common.py", line 595, in get_index_trap
    return get_index(*args, **kwargs)
  File "/opt/wakari/anaconda/lib/python2.7/site-packages/conda/api.py", line 43, in get_index
    unknown=unknown)
  File "/opt/wakari/anaconda/lib/python2.7/site-packages/conda/utils.py", line 118, in __call__
    value = self.func(*args, **kw)
  File "/opt/wakari/anaconda/lib/python2.7/site-packages/conda/fetch.py", line 259, in fetch_index
    repodatas.append((url, future.result()))
  File "/opt/wakari/anaconda/lib/python2.7/site-packages/concurrent/futures/_base.py", line 403, in result
    return self.__get_result()
  File "/opt/wakari/anaconda/lib/python2.7/site-packages/concurrent/futures/thread.py", line 55, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/opt/wakari/anaconda/lib/python2.7/site-packages/conda/fetch.py", line 65, in func
    res = f(*args, **kwargs)
  File "/opt/wakari/anaconda/lib/python2.7/site-packages/conda/fetch.py", line 103, in fetch_repodata
    cache = json.loads(bz2.decompress(resp.content).decode('utf-8'))
IOError: invalid data stream
```

**Comment by bkreider** [comment]
I read that wrong.  That might be coming from the AE-repo, not the config file.

**Comment by bkreider** [comment]
Related to #652

At the very least, conda needs to catch the IOError since this should be expected to happen sometimes.

**Comment by kalefranz** [comment]
All of the issues with config should be fixed in 4.2.
